visual-js/replace_pkg_version.sh

```diff
@@ -34,6 +34,7 @@ FILE_MAP=(
     ["@saucelabs/nightwatch-sauce-visual-service"]="visual-nightwatch/src/utils/constants.ts"
     ["@saucelabs/visual-storybook"]="visual-storybook/src/api.ts"
     ["@saucelabs/wdio-sauce-visual-service"]="visual-wdio/src/SauceVisualService.ts"
+    ["@saucelabs/visual-snapshots"]="visual-snapshots/src/api/client.ts"
```

Comment:
You still need to update path.
```suggestion
    ["@saucelabs/visual-snapshots"]="visual-snapshots/src/index.ts"
```